lib/wallet/rpc.js

```diff
@@ -2665,9 +2665,11 @@ class RPC extends RPCBase {
     const nameHash = rules.hashName(name);
     const address = parseAddress(addr, this.network);
 
-    const blind = await wallet.generateBlind(nameHash, address, value);
+    const blinds = await wallet.generateBlinds(nameHash, address, value);
 
-    return blind.toString('hex');
+    // Return only first blind (based on own key)
+    // to stay backward-compatible
+    return blinds[0].toString('hex');
```

Comment:
eh, backwards-compat isnt so precious here, lets
```js
return blinds.map(b => b.toString('hex'));
```

It'll be a breaking change I guess, if you only have one key youll get an array with a single item but I'm ok with that. Just needs to go in the changelog.

Reply:
Now everything (internal methods, RPC, HTTP) all use smallest key and blinds and nonces are arrays. Added to CHANGELOG.